backend/addcorpus/models.py

```diff
@@ -57,6 +57,10 @@ class Corpus(models.Model):
         help_text='whether the configuration of this corpus is determined by a Python '
             'module (some features are only available for Python-based corpora)',
     )
+    date_created = models.DateField(
+        auto_now_add=True,
+        help_text='date on which the corpus was added to the database',
+    )
 
```

Comment:
I see that I cannot actually make a suggestion on unchanged code, but below on the `class Meta:` definition, you could add:
```python
ordering = ["date_created"]
```
instead of explicitly calling `.order_by` in `views.py`. Or are you after being able to sort according to different criteria?

Reply:
No, but I didn't think that ordering by date was relevant in other contexts.